### PR TITLE
fix output log location

### PR DESCRIPTION
**Pull request by apraovjr**
- Change output log location

- fix filename for teamAAD

### CODE REVIEW

File: tools/validate-deployment.sh [hunk 1]
```diff
@@ -60,7 +62,7 @@ if [[ $ipaddress =~ ^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$ ]]; then
         echo "[ERROR] Public key missing when resetting key for ProctorVM for $teamAAD - Subscription $subid - Exiting ..."
     fi
 
-    scp -o StrictHostKeyChecking=no -i $ID_RSA_PRIVATE -r azureuser@$ipaddress:/home/azureuser/team_env/* ./$teamAAD/
+    scp -o StrictHostKeyChecking=no -i $ID_RSA_PRIVATE -r azureuser@$ipaddress:/home/azureuser/logs/* ./$teamAAD/
```
Comment (dcaro) [inline tools/validate-deployment.sh]: This should get the files from the nginx content directory
`scp -o StrictHostKeyChecking=no -i $ID_RSA_PRIVATE -r azureuser@$ipaddress:/home/nginx/contents/* ./$teamAAD/
`

File: tools/validate-deployment.sh [hunk 2]
```diff
@@ -60,7 +62,7 @@ if [[ $ipaddress =~ ^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$ ]]; then
         echo "[ERROR] Public key missing when resetting key for ProctorVM for $teamAAD - Subscription $subid - Exiting ..."
     fi
```
Comment (dcaro) [inline tools/validate-deployment.sh]: kubeconfig file does not have the proper permissions hence you need to chown it.
`             ssh -o StrictHostKeyChecking=no -i $ID_RSA_PRIVATE azureuser@$ipaddress "bash -s" << EOF
                    sudo chown azureuser:azureuser /home/nginx/contents/kubeconfig;
EOF
`

File: tools/validate-deployment.sh (hunk 1 above)
Comment (apraovjr) [reply tools/validate-deployment.sh]: @dcaro so it shouldnt get from `/home/azureuser/logs` but instead from `/nginx/contents` ?

Comment (dcaro) [reply tools/validate-deployment.sh]: No, /home/nginx/contents. the kvstore and kubeconfig files are in that directory.

**Pull request merged**